system/ee/ExpressionEngine/View/addons/index.php

```diff
@@ -2,19 +2,54 @@
 
 <div class="panel panel__no-main-title">
   <div class="tbl-ctrls">
-    <form>
+       <form>
       <div class="panel-heading">
         <div class="app-notice-wrap"><?=ee('CP/Alert')->getAllInlines()?></div>
-        <div class="alert">
-          <div class="alert__icon"><i class="fas fa-info-circle fa-fw"></i></div>
-          <div class="alert__content">
-              <p class="alert__title">Updates Available</p>
-              <p>There are <strong>2</strong> installed add-ons with updates available. <a href="">Show me...</a></p>
-          </div>
-          <a href="" class="alert__close">
-              <i class="fas fa-times alert__close-icon"></i>
-          </a>
-      </div>
+
+        
+          
+            
+
+            <?php if (! empty($updates) && count($updates) == 1 ) : ?>
+              <div id = "alert_banner" class="alert">
+              <div class="alert__icon"><i class="fas fa-info-circle fa-fw"></i></div>
+              <?php
+              echo '<div id = "alert" href="" class="alert__close">';
+              echo '<a onclick="hide()" id = "alert_close" class="fas fa-times alert__close-icon"></a>';
+              echo '</div>';
+              ?>
+                <p class="alert__title">Update Available <br> 
+                There is <strong><?=count($updates)?></strong> installed add-on with updates available. <a onclick = "typeFilter('app-listing__row_update')">Show me...</a></p>
```

Comment:
I don't think `onclick` attribute should be used here. It is preferred to attach event in JS file